docs/software/ce-test-scaling.md

```diff
@@ -4,7 +4,7 @@ How to Run Scalability Tests on a CE
 Introduction
 ------------
 
-This document is intended as a general overview of the process for scalability testing of an OSG CE (Compute Element). All examples are for testing an [HTCondor-CE](https://twiki.grid.iu.edu/bin/view/Trash/Trash/Sandbox/HTCondorCEScaleTests), but they should be applicable for other CE software.
+This document is intended as a general overview of the process for scalability testing of an OSG CE (Compute Element). All examples are for testing an [HTCondor-CE](/docs/compute-element/install-htcondor-ce), but they should be applicable for other CE software.
```

Comment:
Same as before, you need the full URL for docs in the https://opensciencegrid.github.io/docs/